main.go

```diff
@@ -10,6 +10,8 @@ import (
 	"golang.org/x/oauth2"
 )
 
+var version = "0.0.0"
```

Comment:
yes, that was the intention (currently in form of `git-<commit SHA>`, logic in `Makefile`)